packages/world/package.json

```diff
@@ -18,25 +18,13 @@
     "./node": "./dist/node.js",
     "./out/*": "./out/*"
   },
-  "typesVersions": {
-    "*": {
-      "index": [
-        "./ts/exports/index.ts"
-      ],
-      "internal": [
-        "./ts/exports/internal.ts"
-      ],
-      "config/v2": [
-        "./ts/config/v2/index.ts"
-      ],
-      "register": [
-        "./ts/register/index.ts"
-      ],
-      "node": [
-        "./ts/node/index.ts"
-      ]
-    }
-  },
+  "files": [
+    "dist",
+    "out",
+    "foundry.toml",
+    "cache",
+    "test/MudTest.t.sol"
```

Comment:
@alvrs I wonder if we should move this into src

Reply:
Changing the import path would be a breaking change, would prefer to avoid unless there is a strong reason to do so (but we could add a reexport from src if you feel strongly about importing it from there)